modules/Core/pages/register.php

```diff
@@ -353,6 +353,7 @@
                                     'last_online' => $date,
                                     'language_id' => $language_id,
                                     'active' => $active
+                                    'timezone' => ((isset($_POST['timezone']) && $_POST['timezone']) ? Output::getClean($_POST['timezone']) : Output::getClean('Europe/London'))
```

Comment:
instead of hardcoding `Europe/London`, we have a const of `TIMEZONE` you can replace it with.